packages/client/src/components/UnprotectedRoute.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Spinner } from '@opencrvs/components'
+import { connect } from 'react-redux'
+import { configAnonymousUserLoaded } from '@client/offline/actions'
+import { referenceApi } from '@client/utils/referenceApi'
+import { IStoreState } from '@client/store'
+import { Route } from 'react-router'
+
+const StyledSpinner = styled(Spinner)`
+  position: absolute;
+  margin-left: -24px;
+  margin-top: -24px;
+  top: calc(50% - 20px);
+  left: 50%;
+  width: 40px;
+  height: 40px;
+`
+
+const UnprotectedRouteWrapper = (props: {
+  [x: string]: any
+  configAnonymousUserLoaded: any
```

Comment:
Notice you can also use [useDispatch](https://react-redux.js.org/api/hooks#usedispatch) here so you don't have to use connect and pass action creators in props

```ts
const dispatch = useDispatch()
dispatch(configAnonymousUserLoaded({ config }))
```